styles.css

```diff
@@ -132,3 +133,424 @@ ul {
+  width: 327px;
+  height: 474px;
+  top: 4px;
+  left: 2px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  margin-bottom: 24px;
+  background: #ebf0ee;
+}
+
+.gray-area {
+  height: 252px;
+  width: 327px;
+  background: #ebf0ee;
+}
+
+.rest {
+  margin: auto;
+  background-color: #fff;
+  padding-left: 25px;
+}
+
+#line {
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  padding-right: 25px;
+}
+
+.description {
+  background: #fff;
+  border-radius: 8px;
+  border: 2px solid #ebf0ee;
+  height: 222px;
+}
+
+.post {
+  width: 295px;
+  height: 57px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+  flex: none;
+  order: 0;
+  margin: 16px auto;
+  flex-grow: 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 305px;
+  height: 32px;
+  margin: auto;
+}
+
+.ruby {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  width: 79px;
+  height: 16px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.font {
+  width: 79px;
+  height: 16px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.css {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  width: 21px;
+  height: 16px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.js {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  width: 58px;
+  height: 16px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.html {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  width: 27px;
+  height: 16px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.button-space {
+  width: 305px;
+  height: 101px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: auto;
+  text-align: center;
+  display: flex;
+  align-items: center;
+}
+
+.project-text {
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background: #36b37e;
+  padding: 12px;
+  border: none;
+  border-radius: 4px;
+  margin: auto;
+}
+
+.the-rest {
+  padding-left: 25px;
+}
+
+.about {
+  background-image: url("Images/Aboutme-background\ \(1\).png"), url("Images/illustration.png");
+  background-position: left top, right top;
+  background-repeat: no-repeat;
+}
```

Comment:
I think you did a great job with the changes suggested by the last code reviewer. :+1:  However there are still some things to improve. The background should fit also on the left side of the container as suggested last code reviewer. :eyes: 
![image](https://user-images.githubusercontent.com/98363075/192097846-76db02f0-f580-4580-96c2-a6b434e097ea.png)